cmd/vault-secret-collection-manager/main.go

```diff
@@ -135,6 +137,9 @@ func (c *idNameCache) set(name, id string) {
 
 func (m *secretCollectionManager) mux() *httprouter.Router {
 	router := httprouter.New()
+	router.GET("/", redirectHandler("/secretcollection?ui=true"))
```

Comment:
Why not a separate handler?

Reply:
Mostly simplicity, it needs the same data, it just returns the UI instead of the raw data